docs/guide/snaps-introduction.md

```diff
@@ -0,0 +1,69 @@
+# Snaps Tutorial
+
+## What is Snaps?
+
+Snaps are a new evolution of MetaMask's functionality. By using MetaMask Flask in conjunction with the Snaps API, developers are able to bring their own code to the wallet in a custom-made code laboratory. MetaMask is the first wallet to provide this openness and flexibility to developers, paving the way to scale the number of features and personalization available to the wallet and your Dapp.
+
+Snaps allows developers to build and plug in their own APIs to MetaMask. No longer do you have to rely on MetaMask to roll out features and updates. The rate of innovation, and MetaMask's flexibility, are limited only by your imagination and your capacity to build. 
+
+## MetaMask Snaps: Overview
+
+Broadly speaking, MetaMask Snaps consists of two things: (1) a way to run untrusted JavaScript inside the MetaMask application, and (2) APIs for websites and MetaMask to communicate with individual snaps. As with MetaMask’s [Ethereum JavaScript provider API](https://docs.metamask.io/guide/ethereum-provider.html), communication occurs via JSON-RPC requests and responses.
+
+### Current functionality
+
+At present, snaps can make JSON-RPC requests of MetaMask, and websites can make JSON-RPC requests of MetaMask and snaps running inside MetaMask. In this way, snaps can expand MetaMask’s RPC API and change the behavior of the MetaMask application at runtime. Over time, MetaMask will use this to support different blockchains and other decentralized protocols, novel kinds of cryptography, novel kinds of crypto assets, and a countless number of other features.
```

Comment:
This makes it seem like the ability for snaps to extend the RPC API is what will enable decentralized protocols, novel cryptography, etc. That's not really true. Primarily it will be us building snap APIs that allow customizing the wallet experience that will enable that stuff. Extending the RPC API is just one thing a snap can do among many things.

Also we should probably not be framing "novel kinds of cryptography" as a future feature. We will enable that at launch.

Reply:
Rather than updating this to be consistent with the reference docs, maybe we can delete this entire section and refer there instead?

Reply:
Or if we want to keep it, here's my current draft:

>At present, snaps can create new RPC methods for websites to call, they can call many of the same RPC methods that websites can call, and they have access to a limited set of snap-exclusive RPC methods (see the Snaps RPC API documentation for more information). Over time, MetaMask will expose more internal functionality as RPC methods, granting more capabililities to snaps.

Reply:
I think deleting and referring the users directly to the Snaps API section of the MM RPC API docs sounds sensible. No need to frame things differently in different places.

That is what you mean by 'refer there instead', right?

Reply:
Yep! OK.